packages/react-composites/tests/browser/call/hermetic/RoomsCallScreen.test.ts

```diff
@@ -71,3 +79,86 @@ test.describe('Rooms CallScreen tests for different roles', async () => {
     );
   });
 });
+
+test.describe('Rooms Participant RemoveButton tests for different roles', async () => {
+  test('Remove button is enabled for Presenter', async ({ page, serverUrl }, testInfo) => {
+    test.skip(isTestProfileStableFlavor());
+    const paul = defaultMockRemoteParticipant('Paul Bridges');
+    const participants = [paul];
+    const initialState = defaultMockCallAdapterState(participants);
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState, { role: 'Presenter' }));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+
+    if (!isTestProfileDesktop(testInfo)) {
+      await pageClick(page, dataUiId('call-with-chat-composite-more-button'));
+      const drawerPeopleMenuDiv = await page.$('div[role="menu"] >> text=People');
+      await drawerPeopleMenuDiv?.click();
+      await hidePiPiP(page);
+      const participantList = await waitForSelector(page, dataUiId('participant-list'));
+      const participantItem = await participantList.$('div[role="menuitem"]');
+      await participantItem?.click();
```

Comment:
After clikcing on the participant item, but before taking the snapshot, you should wait for the drawer with the remove button to show. You need a `waitForSelector` here to avoid flaky test.

Reply:
done